code/modules/jobs/job_types/job.dm

```diff
@@ -147,6 +147,8 @@
 /datum/job/proc/map_check()
 	return TRUE
 
+/datum/job/proc/radio_help_message(mob/M)
+	to_chat(M, "<b>To speak on your departments radio, use the :h button. To see others, look closely at your headset.</b>")

```

Comment:
There isn't a ":h" button. Should say something like: _prefix your message with ":h:" to speak on your department's radio_

Reply:
Yeah, changed wording a bit here.